src/media/css/elements/nav/index.styl

```diff
@@ -0,0 +1,212 @@
+@require('../../constants/typography');
+@require('constants');
+
+mkt-nav {
+  border-right: 2px solid rgba(0, 0, 0, 0.2);
+  fixed: top 0 left 0;
+  font-size: 18px;
+  height: 100%;
+  overflow-y: scroll;
+  transition: transform cubic-bezier(0.86, 0, 0.07, 1) 333ms;
+  width: 260px;
+  z-index: $zindex--nav-hamburger-root;
+
+  li {
+    margin: 25px 0;
+  }
+
+  a {
```

Comment:
A single class has a higher specificity than 2 (or an infinite number of?) tags but in the "allow someone to easily override this rule" sense it won't make a difference. Whoever overrides this will need to specify whatever is in this file.

You can't override the current selector with just `a` for obvious reasons so you'll either use `mkt-nav a` or `mkt-nav.my-nav a`, depending on if you want extra scoping or you load your rules before these ones.

With a class you can do `.mkt-nav--link` or `.my-nav .mkt-nav--link`. So in practice tag vs class isn't going to make a difference, it will be easy to override. I think a class is easier to reason about and results in less footguns and generally speaking using classes is a Good Idea.